### PR TITLE
Updating OneSignal Partner Source Documentation

### PR DESCRIPTION
**Pull request by fhboswell**
### Proposed changes
- Updating events and properties in the OneSignal source doc

### Merge timing
- ASAP once approved

### CODE REVIEW

File: src/connections/sources/catalog/cloud-apps/onesignal/index.md [hunk 1]
```diff
@@ -47,40 +47,46 @@ OneSignal doesn't send user email address or phone number details to Segment.
 
 The following table lists the events that OneSignal sends to Segment. These events appear as tables in your warehouse and as regular events in other destinations. 
 
-| Event Kind                    | Event Description                                    |
+| MessageEvent Kind             | Event Description                                    |
```
Comment (tcgilbert) [inline src/connections/sources/catalog/cloud-apps/onesignal/index.md:50]:
```suggestion
| Message Event Type             | Event Description                                    |
```

**Review by tcgilbert — CHANGES_REQUESTED** [review]

File: src/connections/sources/catalog/cloud-apps/onesignal/index.md (hunk 1 above)
Comment (tcgilbert) [reply src/connections/sources/catalog/cloud-apps/onesignal/index.md:50]: @fhboswell just requested one change. If "kind" is specific to your platform you can keep that over "type"

**Comment by tcgilbert** [comment]
@rchinn-segment I'm currently seeing `object cloud source` for this source. I believe this is because the last time this page was built the integration was hidden. 
Screenshot 2023-12-27 at 11 09 53 AM

Running `make catalog` corrects for this locally, but just want to ensure this is corrected when merged and deployed
Screenshot 2023-12-27 at 11 11 04 AM

File: src/connections/sources/catalog/cloud-apps/onesignal/index.md [hunk 2]
```diff
@@ -47,40 +47,46 @@ OneSignal doesn't send user email address or phone number details to Segment.
...
+| `messageId`                | The identifier of the discrete message                                                |
+| `campaign_id`              | The identifier of the message campaign                                                |
+| `message_name`             | The message name                                                                      |
+| `message_title`            | The message title only for push and email                                             |
+| `message_contents`         | The message contents only for push and sms                                            |
+| `template_id`              | The message template used                                                             |
+| `subscription_id`         | The OneSignal set device/email/sms identifier                                         |
+| `subscription_device_type` | The device type that received the message                                             |
+| `language`                 | The two character language code of the device                                         |
+| `message_type`             | The type of message sent, push, in-app, email, SMS                                    |
```
Comment (rchinn-segment) [inline src/connections/sources/catalog/cloud-apps/onesignal/index.md:89]:
```suggestion
| `message_type`             | The type of message sent (push, in-app, email, or SMS)                                    |
```

**Review by rchinn-segment — CHANGES_REQUESTED** [review]

**Comment by rchinn-segment** [comment]
@tcgilbert - Thanks for calling this out. We'll run `make catalog` before deploying this along with the rest of the docs

File: src/connections/sources/catalog/cloud-apps/onesignal/index.md (hunk 1 above)
Comment (fhboswell) [reply src/connections/sources/catalog/cloud-apps/onesignal/index.md:50]: Hey @tcgilbert we use "kind" in our platform.

**Review by tcgilbert — APPROVED** [review]
looks good 👍

**Pull request merged**